simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountFragment.kt

```diff
@@ -409,13 +410,22 @@ class AccountFragment : Fragment() {
       if (cardCreator == null) {
         this.logger.error("Card creator not configured")
       } else {
-        cardCreator.openCardCreatorActivity(
-          this,
-          this.activity,
-          this.cardCreatorResultCode,
-          this.account.loginState is AccountLoggedIn,
-          this.authenticationBasicUser.text.toString().trim()
-        )
+        // Launch NYPL card creator
+        if (this.account.provider.cardCreatorURI?.scheme.equals(nyplCardCreatorScheme)) {
+          cardCreator.openCardCreatorActivity(
+            this,
+            this.activity,
+            this.cardCreatorResultCode,
+            this.account.loginState is AccountLoggedIn,
+            this.authenticationBasicUser.text.toString().trim()
+          )
+          // Launch external card creator
+        } else {
```

Comment:
The logic for this is a bit wrong. The actual card creator and web calls look correct, but the branches should look more like:

```
// Does the library require the NYPL card creator?
if (cardCreatorURI.scheme.equals(nyplCardCreatorScheme)) {
  // Does this build actually have an NYPL card creator?
  if (cardCreator != null) {
    // Open NYPL card creator
  }
} else {
  // Open web card creator instead
}
```
There needs to be similar logic to check if the button should be enabled/disabled.